client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -46,9 +47,12 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <InputLabel htmlFor="email">
+            <Typography className={classes.label}>E-MAIL ADDRESS</Typography>
```

Comment:
Reserve the use of capital letters for abbreviations (e.g. `CSS`) for accessibility reasons, instead use CSS to transform the case of a text when necessary. ([ref](https://medium.com/@mandy.michael/why-you-shouldnt-write-your-content-in-uppercase-instead-use-css-b03ac2c65b99))

Ditto throughout 👍🏼